glean-core/android/src/test/java/mozilla/telemetry/glean/TestUtil.kt

```diff
@@ -18,8 +16,6 @@ import kotlinx.coroutines.isActive
 import kotlinx.coroutines.runBlocking
 import kotlinx.coroutines.withTimeout
 import org.json.JSONObject
-import org.mockito.ArgumentMatchers
-import org.mockito.Mockito
```

Comment:
You probably need to keep this to fix the tests.

Reply:
screw the tests! (yeah, I removed the wrong line and didn't run it again locally -_-)